RecoEgamma/EgammaTools/plugins/BuildFile.xml

```diff
@@ -3,6 +3,7 @@
   <use   name="FWCore/Framework"/>
   <use   name="PhysicsTools/MVAComputer"/>
   <use   name="PhysicsTools/MVATrainer"/>
+  <use   name="PhysicsTools/PatAlgos"/>
```

Comment:
if you move ModifyObjectValueBase to CommonTools subsystem (as suggested above, the issue goes away)